lib/workers/repository/config-migration/branch/migrated-data.ts

```diff
@@ -142,10 +143,10 @@ export class MigratedDataFactory {
       delete migratedConfig.errors;
       delete migratedConfig.warnings;
 
-      // indent defaults to 2 spaces
       // TODO #22198
-      const raw = await readLocalFile(configFileName!, 'utf8');
+      const raw = await platform.getRawFile(configFileName!);
       const indent = detectIndent(raw!);
```

Comment:
```suggestion
       const indent = detectIndent(raw ?? '');
```
we should be more defensive here